### PR TITLE
grafana-agent 0.24.0

### PR DESCRIPTION
**Pull request by grafanabot**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by rlankfo** [comment]
I'm not sure what is causing this failure:
https://github.com/Homebrew/homebrew-core/runs/5877253138?check_suite_focus=true

I'll look into this and see if I can determine what the problem is.

**Comment by rlankfo** [comment]
thanks :pray: for adding the build patch @chenrui333, I wasn't aware I could do this. I've submitted another PR to fix the build issues: https://github.com/Homebrew/homebrew-core/pull/99058. I'll rebase this PR once it's merged.

**Comment by chenrui333** [comment]
I can include your another commit change as well.

**Comment by chenrui333** [comment]
> thanks 🙏 for adding the build patch @chenrui333, I wasn't aware I could do this. I've submitted another PR to fix the build issues: #99058. I'll rebase this PR once it's merged.

sounds good!

File: Formula/grafana-agent.rb [hunk 1]
```diff
@@ -21,6 +21,12 @@ class GrafanaAgent < Formula
     depends_on "systemd" => :build
   end
 
+  # Remove with next release.
```
Comment (SMillerDev) [inline Formula/grafana-agent.rb:24]: What does it do? What release is supposed to be the next release?

**Comment by chenrui333** [comment]
#99290

**Pull request closed without merging**